### ISSUE TITLE
Refresh Skype window after (un)check "Hide ..." options

### ISSUE THREAD

**Issue by nestdimon** [dlehn/Skype-Utility-Project#6]
Here is Skype main window before applying SUP layout options:
![before](https://cloud.githubusercontent.com/assets/5602016/5691055/29f3a0f8-98c5-11e4-8731-108030490dae.png)

And here is immediately after:
![after](https://cloud.githubusercontent.com/assets/5602016/5691056/2cb1cdce-98c5-11e4-89b4-7a8922782e88.png)

It looks like SUP must force Skype to recalculate controls position and repaint window.

**Comment by dlehn** [comment]
Yes, unfortunately that's known problem and is mentioned in the readme. Skype will not adapt to any layout changes until you select another conversation (Combined View) or resize the contact list window (Split View).  
I already tried forcing the window to repaint without success. I'll see if I can find some other way around it. Maybe sending a `WM_SIZE` event will work. I won't be able to work on this until tonight, but I'll report back once I know more.

**Comment by nestdimon** [comment]
Ok, I'll try too.

**Comment by nestdimon** [comment]
All what is need to do:

```
WINDOWPOS pos; // Even not filled works
SendMessage(_hwnd, WM_WINDOWPOSCHANGED, NULL, (LPARAM) &pos);
```

after `hideAppToolbarChanged()` and `hideIdentityPanelChanged()`.

I've send a pull request.

**Comment by dlehn** [comment]
Does this always work for you?
When I test it, it will update fine when changing the options from hide to show, but not the other way round.

Edit: Nevermind, that was my fault. I messed up when refactoring your change,

**Comment by dlehn** [comment]
Thanks a lot for the fix!  
Merged your code with some changes, see #7.
